src/main/java/org/comic_conmuseum/fan_forge/backend/persistence/SupportQueryBean.java

```diff
@@ -34,8 +40,9 @@ public void setupTable(boolean reset) {
                 "    sid SERIAL PRIMARY KEY, " +
                 "    exhibit SERIAL REFERENCES exhibits(eid) ON DELETE CASCADE ON UPDATE CASCADE, " +
                 "    supporter TEXT ,"+//TODO SERIAL REFERENCES users(uid) ON DELETE CASCADE ON UPDATE CASCADE, " +
-                // TODO Get actual survey data fields to use and use them
-                "    survey_data TEXT, " +
+                "    visits INTEGER NOT NULL CHECK (0 <= visits AND visits <= 10), " +
+                "    rating INTEGER NOT NULL CHECK (0 <= rating AND rating <= 10), " +
+                "    " + POPULATIONS_COLUMN_DEFS + ", " +
```

Comment:
From the API docs:
visits: integer // [1-10] How many times the respondent would visit this exhibit
rating: integer // [1-10] Rating given by the user, for NPS calculation

I think you should change these to be 0 < visits, or 1 <= visits. Same with rating

Reply:
Whoops, I think I made the mistake in the API docs there, actually. I'll fix that.